src/constants.ts

```diff
@@ -17,3 +17,9 @@ export const CHILD_REGISTER_ENCOUNTER_TYPE = "";
 //maternity
 export const ADMISSION_DATE = "cb9b223e-a09c-4f8a-b34d-a0934a2af04a";
 export const DELIVERY_TYPE = "dcc3ac63-30ab-102d-86b0-7a5022ba4115";
+
+// opd
+export const OUTPATIENT_REGISTER_ENCOUNTER_TYPE =
+  "ee4780f5-b5eb-423b-932f-00b5879df5ab";
+export const REFARRAL_NOTE_ENCOUNTER_TYPE =
```

Comment:
@Irenyak1 check the spelling of  referral, also where is this UUID coming from? it seems not to exist in out database

Reply:
@solemabrothers  sure let me fix the spelling. The UUID is a place holder since many UUIDs have not been provided to us yet.

Reply:
@Irenyak1 work with @Mwanje to get you this UUID, there is already an existing UUID for the referral encounter.

Reply:
@solemabrothers sure I am going to get to him. I am in a black out and my laptop's battery is running out.

Reply:
@solemabrothers  @Mwanje @slubwama  any more reviews on this?